core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -243,6 +243,108 @@ describe('Core exploration functionality', function() {
     users.logout();
   });
 
+  it('should be able to edit title', function() {
+    users.createUser('user6@stateEditor.com', 'user6StateEditor');
+    users.login('user6@stateEditor.com');
+
+    workflow.createExploration();
```

Comment:
Can we do the above 3 statements (which is common setup), in a beforeAll block? That way it needs to run only once.

Reply:
Hi @nithusha21, thanks for reviewing! I tried adding it to beforeAll block just now, but I found some problems with it. If we only create an exploration once, sometimes it's an old exploration.  It's better to test it with a new exploration because when it's old there is something redundant that we're not expecting. And the same situation for logging in.

Reply:
Sorry for the delay! I understand, if you want to redo it everytime, you can use a beforeEach block (runs before each it block). That's better than duplicating the code.